crates/api_common/src/send_activity.rs

```diff
@@ -40,7 +35,7 @@ pub enum SendActivityData {
   CreatePost(Post),
   UpdatePost(Post),
   DeletePost(Post, Person, DeletePost),
-  RemovePost(Post, Person, RemovePost),
+  RemovePost(Post, Person, Option<String>, bool),
```

Comment:
Don't need to do in this PR, but it'd be nice if these were all converted to named field structs like `BanFromSite`, then it'd be much more clear what `Option<String>` means. Maybe add a TODO if that's something you want to do.

Reply:
Ive converted all with 4+ params, the remaining ones should be simple enough. Plus it would be extremely verbose to use named params everywhere.